Sources/LyftUI/LyftButton.swift

```diff
@@ -60,7 +60,7 @@ public class LyftButton: UIView {
         self.init(frame: CGRect(origin: CGPoint.zero, size: LyftButtonPreferredSize))
     }
 
-    public override required init?(coder: NSCoder) {
+    public required init?(coder: NSCoder) {
```

Comment:
do you need this at all? since the other one is a `convenience` I don't think you need this method.